remote-processor/Message.cpp

```diff
@@ -28,51 +28,58 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 #include "Message.h"
+#include "Iterator.hpp"
 #include <vector>
+#include <numeric>
+#include <cassert>
 
-using std::string;
+using namespace std;
```

Comment:
Pretty sure this change is not needed. All std algorithm will be found be [ald](http://en.cppreference.com/w/cpp/language/adl).